src/classes/BDI_FieldMapping.cls

```diff
@@ -89,6 +89,13 @@ public virtual with sharing class BDI_FieldMapping {
             this.Target_Object_API_Name,
             this.Target_Field_API_Name);
 
+        //Updating field and object names to make sure they are using the actual correctly cased
+        //API names so it doesn't cause problems with some LWC functions
+        this.Target_Object_API_Name = UTIL_Describe.getObjectName(this.Target_Object_API_Name);
+        this.Target_Field_API_Name = UTIL_Describe.getFieldName(this.Target_Object_API_Name, this.Target_Field_API_Name);
```

Comment:
Style update only, but can we avoid the long line and add a linebreak in here?

Reply:
I don't think we can do a line break here. It doesn't look excessively long in VSCode.